Recent change to AggregatingReducer breaks ingest

While testing locally, I found that [this change](https://github.com/NationalSecurityAgency/datawave/blob/186b2deb1e8cb401c9e80ee288845078d7476630/warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/AggregatingReducer.java#L381) breaks ingest, (see stacktrace below). The exception is due to the `isValidEncoding(columns)` logic within `ColumnSet.decodeColumns(column)` b/c what we pass in is a literal '*' in this case and thatis deemed to be invalid by `isValidEncoding`

Still investigating the ideal fix. Will also investigate adding unit test coverage for this case
```
19/04/30 16:25:52 INFO mapreduce.Job: Task Id : attempt_1556641296382_0003_m_000000_2, Status : FAILED
Error: java.io.IOException: Failed to initialized class datawave.ingest.mapreduce.job.writer.DedupeContextWriter from property ingest.event.mapper.context.writer.class
        at datawave.ingest.mapreduce.EventMapper.setup(EventMapper.java:245)
        at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:142)
        at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:787)
        at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
        at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:164)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:422)
        at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1912)
        at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)
Caused by: java.lang.IllegalArgumentException: Invalid encoding *
        at org.apache.accumulo.core.iterators.conf.ColumnSet.decodeColumns(ColumnSet.java:126)
        at datawave.ingest.mapreduce.job.reduce.AggregatingReducer$CustomColumnToClassMapping.<init>(AggregatingReducer.java:381)
        at datawave.ingest.mapreduce.job.reduce.AggregatingReducer.configureAggregators(AggregatingReducer.java:256)
        at datawave.ingest.mapreduce.job.reduce.AggregatingReducer.configureReductionInterface(AggregatingReducer.java:125)
        at datawave.ingest.mapreduce.job.reduce.AggregatingReducer.setup(AggregatingReducer.java:107)
        at datawave.ingest.mapreduce.job.reduce.BulkIngestKeyDedupeCombiner.setup(BulkIngestKeyDedupeCombiner.java:38)
        at datawave.ingest.mapreduce.job.writer.DedupeContextWriter.setup(DedupeContextWriter.java:35)
        at datawave.ingest.mapreduce.EventMapper.setup(EventMapper.java:243)
        ... 8 more
```

--- a/contrib/datawave-quickstart/bin/services/datawave/bootstrap-web.sh
+++ b/contrib/datawave-quickstart/bin/services/datawave/bootstrap-web.sh
@@ -14,7 +14,7 @@ DW_DATAWAVE_WEB_CMD_START="( cd "${WILDFLY_HOME}/bin" && JAVA_OPTS=\"$DW_DATAWAV
 DW_DATAWAVE_WEB_CMD_START_DEBUG="( cd "${WILDFLY_HOME}/bin" && JAVA_OPTS=\"$DW_DATAWAVE_WEB_JAVA_OPTS\" nohup ./standalone.sh --debug -c standalone-full.xml & )"
 DW_DATAWAVE_WEB_CMD_STOP="datawaveWebIsRunning && [[ ! -z \$DW_DATAWAVE_WEB_PID_LIST ]] && kill -15 \$DW_DATAWAVE_WEB_PID_LIST"
 
-DW_DATAWAVE_WEB_CMD_FIND_ALL_PIDS="pgrep -f 'jboss.home.dir=${WILDFLY_HOME}'"
+DW_DATAWAVE_WEB_CMD_FIND_ALL_PIDS="pgrep -f 'jboss.home.dir=${DW_CLOUD_HOME}/*wildfly'"
 
 DW_DATAWAVE_WEB_SYMLINK="datawave-webservice"
 DW_DATAWAVE_WEB_BASEDIR="datawave-webservice-install"

--- a/warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/AggregatingReducer.java
+++ b/warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reduce/AggregatingReducer.java
@@ -365,7 +365,8 @@ public abstract class AggregatingReducer<IK,IV,OK,OV> extends Reducer<IK,IV,OK,O
      * Helper class that, given a {@link Key}, determines which aggregator, if any, should be used to aggregate multiple values for that key.
      */
     protected static class CustomColumnToClassMapping extends ColumnToClassMapping<Combiner> implements Comparable<CustomColumnToClassMapping> {
-        protected static Key ALL_CF_KEY = new Key("", "*");
+        private static final String ALL_CF_STR = "*";
+        protected static final Key ALL_CF_KEY = new Key("", ALL_CF_STR);
         protected Integer priority;
         
         public CustomColumnToClassMapping(Integer priority, Map<String,String> opts) {
@@ -378,7 +379,12 @@ public abstract class AggregatingReducer<IK,IV,OK,OV> extends Reducer<IK,IV,OK,O
                 
                 final String className = entry.getValue();
                 
-                Pair<Text,Text> pcic = ColumnSet.decodeColumns(column);
+                Pair<Text,Text> pcic;
+                if (ALL_CF_STR.equals(column)) {
+                    pcic = new Pair<>(ALL_CF_KEY.getColumnFamily(), null);
+                } else {
+                    pcic = ColumnSet.decodeColumns(column);
+                }
                 
                 Combiner agg = null;
                 
@@ -412,7 +418,12 @@ public abstract class AggregatingReducer<IK,IV,OK,OV> extends Reducer<IK,IV,OK,O
                 
                 final String className = clazzOptions.getValue();
                 
-                Pair<Text,Text> pcic = ColumnSet.decodeColumns(column);
+                Pair<Text,Text> pcic;
+                if (ALL_CF_STR.equals(column)) {
+                    pcic = new Pair<>(ALL_CF_KEY.getColumnFamily(), null);
+                } else {
+                    pcic = ColumnSet.decodeColumns(column);
+                }
                 
                 Combiner agg = null;
                 

